src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -14425,17 +14448,20 @@ Map {
             "stateReducer": Object {
               "type": "func",
             },
+            "suppressRefError": Object {
+              "type": "bool",
+            },
           },
         ],
         "type": "shape",
       },
       "editOptionText": Object {
         "type": "string",
       },
-      "findHighlightedIndex": Object {
-        "type": "func",
-      },
```

Comment:
Was previously used when we copied ComboBox over to CarbonComboBox. Refactored ComboBox no longer needs this override. CarbonComboBox was removed, we now use the stock component from Carbon.